src/Common/Query/GeocodeQuery.php

```diff
@@ -46,12 +46,12 @@
     /**
      * @var array
      */
-    private $data = [];
```

Comment:
Why do you need to modify the scope of `$data`? You could just introduce a new property on `Geocoder\Provider\MapQuest\GeocodeQuery`, right?

Reply:
My goal was that the new `GeocodeQuery` class could be a drop in and exchangeable with the old one. If I introduce a new property, I would also have to reimplement the withData method right?

It seemed easier for me to do it this way. If you agree to remove the `final` keyword, why cannot all properties be protected?